Sources/TuistSupport/Utils/FileHandler.swift

```diff
@@ -226,3 +244,61 @@ public class FileHandler: FileHandling {
         }
     }
 }
+
+// Need to find a suitable place for XCFrameworkInfoPlist
+
+public struct XCFrameworkInfoPlist: Codable {
```

Comment:
I don't think this should be in `TuistSupport`. What about moving it to `TuistCore` where all the Tuist models live?

Reply:
we could move it to `TuistCore`